sklearn/feature_selection/rfe.py

```diff
@@ -184,14 +190,20 @@ def _fit(self, X, y, step_score=None):
             estimator.fit(X[:, features], y)
 
             # Get coefs
-            if hasattr(estimator, 'coef_'):
-                coefs = estimator.coef_
+            if callable(self.importance_getter):
+                coefs = self.importance_getter(estimator)
+            elif self.importance_getter=='auto':    
+                if hasattr(estimator, 'coef_'):
+                    coefs = estimator.coef_
+                else:
+                    coefs = getattr(estimator, 'feature_importances_', None)
+                if coefs is None:
+                    raise RuntimeError('The classifier does not expose '
+                                    '"coef_" or "feature_importances_" '
```

Comment:
Please fix indentation

Reply:
Sure. Sorry for push without flake8 checking.